docs/research/index.md

```diff
@@ -0,0 +1,43 @@
+* practical forms of session management
+* awareness of what the other participants are doing
+
+We are studying these aspects via multiple channels such as gathering written user feedback, collecting and analyzing fine-grained usage data, and explicit usability studies.
+ 
+### Saros usage statistics and diagrams 
+Saros includes a statistical framework which is responsible for gathering information about the actual usage. This information ranges from the plain duration of a session to much more detailed facets such as role swaps performed or text edits contributed. For each session—assuming one allows the submission of this anonymous information—a session statistic is sent to our server.
+
+**The statistical framework is disabled in the current release**, because the corresponding infrastructure to process the gathered data is not currently set up.
+
+## Publications on Saros
+
+### Reviewed research articles
+* Lutz Prechelt and Karl Beecher. “[Four generic issues for tools-as-plugins illustrated by the distributed editor Saros.](http://dl.acm.org/citation.cfm?id=1984708.1984712)” In Proceeding of the 1st workshop on Developing tools as plug-ins, 9. ACM Press, 2011.
+* Lutz Prechelt. “[Some non-usage data for a distributed editor.](http://dl.acm.org/citation.cfm?id=1984642.1984651)” In Proceeding of the 4th international workshop on Cooperative and human aspects of software engineering, 48. ACM Press, 2011.
+* Edna Rosen, Stephan Salinger, and Christopher Oezbek. “[Project Kick-off with Distributed Pair Programming.](http://www.ppig.org/papers/22nd-SE-1.pdf)” In Proceedings of the 22nd Annual Workshop of Psychology of Programming Interest Group, 121-135. Leganès, Spain: Universidad Carlos III de Madrid, 2010.
+* Stephan Salinger, Christopher Oezbek, Karl Beecher, and Julia Schenk. “[Saros: an eclipse plug-in for distributed party programming.](https://dl.acm.org/citation.cfm?id=1833319)” In Proceedings of the 2010 ICSE Workshop on Cooperative and Human Aspects of Software Engineering, 48-55. ACM Press, 2010.
+
+### Unreviewed research articles
+* Riad Djemili, Christopher Oezbek, and Stephan Salinger. “[Saros: Eine Eclipse-Erweiterung zur verteilten Paarprogrammierung.](http://www.inf.fu-berlin.de/inst/ag-se/pubs/saros-2007.pdf)” In Software Engineering 2007 - Beiträge zu den Workshops, P-106:317-320. GI-Edition - Lecture Notes in Informatics. Bonn: Köllen Verlag, 2007.
+
+### Theses
+The working group software engineering also oversees bachelor's and master's theses in the field of computer science at the Freie Universität Berlin. Many of these theses deal with the topic of (distributed) pair programming and/or Saros. For more information, see the list of [finished theses](http://www.inf.fu-berlin.de/w/SE/ThesesHome#Abgeschlossene_Arbeiten) and [theses in progress](http://www.inf.fu-berlin.de/w/SE/ThesesHome#Laufende_Arbeiten).
+* See [finished theses](http://www.inf.fu-berlin.de/w/SE/ThesesHome#Abgeschlossene_Arbeiten)
```

Comment:
I don't think you need the explicit listings here as they are already mentioned in the text. But you can still keep them around if you want to make the links easier to find.